py/torch_tensorrt/dynamo/_tracer.py

```diff
@@ -80,8 +76,8 @@ def trace(
                         max=max_shape[dim],
                     )
 
-            dynamic_shapes[input.name] = dynamic_dims
+            dynamic_shapes.append(dynamic_dims)
```

Comment:
What is this list aligned against? the inputs to the module or the trt inputs? Is there a way to make this a bit more legible?

Reply:
This list is aligned against the user provided inputs (which are inputs to the module and their respective dynamic shapes if present). Pytorch provides 2 options either a dictionary with names or a tuple of dynamic shapes. We used to do the former but the problem with that in general is user needs to know the variable names in Pytorch which is not trivial in some cases.